packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -247,6 +247,22 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     return TestAsyncUtils.guard<void>(() => binding.pump(duration, phase));
   }
 
+  /// Similar to [pump], [pumpBenchmark] triggers a frame to be drawn; however,
+  /// it can be used when the frame policy is set to  [LiveTestWidgetsFlutterBindingFramePolicy.benchmark]
+  /// to artificially drive a high CPU load by rendering frames in a tight loop.
+  Future<void> pumpBenchmark(Duration duration) async {
+    dynamic caughtException;
```

Comment:
should we assert that the policy is benchmark?

Reply:
It might catch some bugs and I don't see a reason not to.